pages/vi/vi-planetapps.md

```diff
@@ -40,8 +40,10 @@ Next, go to [your community Planet](http://localhost:3100). Select `Library` fro
 
 Fill out the information:
 
-- As we are just doing this as a practice, accuracy of information/source of content is not very important.
-- Make sure to have something in all of the drop-down menus and text boxes.
+1. Fill in all of the text boxes and drop-down menus (As this is just practice, accuracy of information is not important)
+2. Be sure to **upload** one of the `PDF - "Feelings"`, `mp3 song - "Opposite Song"`, or `mp4 video - "Burka Avenger"` files.
+3. Next, click on Submit. You have now uploaded the resource.
+4. Repeat this until all three files have been uploaded as resources.
 
 Next, click on `Submit`. You have now uploaded the resource. You should be able to find the resource in Library, make sure that you can open it. Repeat the same process for all three resources.
```

Comment:
@samuelchen1213, looks good. However, could you please delete a paragraph after ordered list (line 48) and add one more item between 3 and 4 with information like this - `Find the resource in Library, and make sure that you can open it`.